<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistroOperacion</class>
 <widget class="QDialog" name="RegistroOperacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrar Nueva Operaci贸n</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBoxDatos">
     <property name="title">
      <string>Datos de la Operaci贸n</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelFecha">
        <property name="text">
         <string>Fecha:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDateEdit" name="dateEditFecha"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelTipo">
        <property name="text">
         <string>Tipo de Operaci贸n:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboTipo"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelOrigen">
        <property name="text">
         <string>Origen:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="comboOrigen"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelDestino">
        <property name="text">
         <string>Destino:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboDestino"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelPlaza">
        <property name="text">
         <string>Plaza:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboPlaza"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelTransportadora">
        <property name="text">
         <string>Transportadora:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboTransportadora"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxDenominaciones">
     <property name="title">
      <string>Denominaciones</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="labelValor">
          <property name="text">
           <string>Valor:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinValor">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelCantidad">
          <property name="text">
           <string>Cantidad:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinCantidad">
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAgregarDenominacion">
          <property name="text">
           <string>Agregar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableWidget" name="tableDenominaciones">
        <column>
         <property name="text">
          <string>Valor</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Cantidad</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Subtotal</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTotal">
     <property name="text">
      <string>Total: $0.00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight</set>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
       <pointsize>12</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnRegistrar">
       <property name="text">
        <string>Registrar Operaci贸n</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>